app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2525,9 +2528,16 @@ class BrowserTabFragment :
     }
 
     private fun launchAutofillManagementScreen() {
+        val privacyProtectionStatus = when (viewModel.privacyShieldViewState.value?.privacyShield) {
```

Comment:
Can we make it that you receive this value as part of the command? That way viewmodel is the one accessing that value, not the view.

Reply:
As discussed offline, changed how this value is determined. Thanks!